controller/applicationsController.js

```diff
@@ -0,0 +1,52 @@
+const Applications = require("../models/applications");
+const mongoose = require("mongoose");
+const CustomError = require("../utils/customError");
+const responseHandler = require("../utils/responseHandler");
+const Organizations = require("../models/organizations");
+
+exports.getSingleApplication = async (req, res, next) => {
+  try {
+    const { organizationId } = req.params;

```

Comment:
Please check the sheets and online API docs. The route here should be GET `/applications/:applicationId`. The `organizationId` is available on the `req.token` not the params.